installers/cubeit-installer

```diff
@@ -854,10 +854,10 @@ if [ -n "${HDINSTALL_CONTAINERS}" ]; then
 
 	case $container_src in
 	    *-dom0-*)
-		${SBINDIR}/cube-ctl add --auto essential -n ${cname} -t cube -o ${CONTAINERBASE} ${c}
+		${SBINDIR}/cube-ctl add --track --auto essential -n ${cname} -t cube -o ${CONTAINERBASE} ${c}
```

Comment:
Well done in picking up the change I had to make in cube-ctl for this. I was going to email today about it.

I was seeing [ERROR] ... messages during on target install from overc-ctl and wasn't able to root cause the issue, so I protected the calls with that new flag.

Obviously you weren't seeing those same errors, or didn't get a chance to try it on target yet. But if you can try adding a tracked container on target .. it would be good to confirm that it is now clean.